lib/nrf5/common/spim_common.c

```diff
@@ -0,0 +1,41 @@
+/** @addtogroup spim_file
+
+@author @htmlonly &copy; @endhtmlonly 2018 
+Gerrit Maus <funk@maus.xyz>
+Luong Le <novercy@live.com>
+
+This library supports the SPIM in the NRF52 series.
```

Comment:
you have more room here, use it. don't just write comments that _appear_ to be commnets, but don't actually say anything.